conf/modules.config

```
@@ -394,7 +397,7 @@ process {
 
     withName: MODKIT_PILEUP_POPULATION_SCALE {
         ext.args = { [
-            '--cpg',
+            params.population_context ? "-k --${params.population_context}" : '',
```

so by default, methylation will be called from the CpG context
```suggestion
            params.all_contexts ? '--motif CHH 0 --motif CHG 0 --motif CG 0' : '--motif CG 0',  
            params.m6a ? '--motif A 0' : '',  
```

---

yess I left a comment there too, also for `MODKIT_PILEUP`

---

okay, then it should be documented somewhere, probably in readme.md